TipCatDotNet.Api/Services/Payments/TransactionService.cs

```diff
@@ -197,10 +177,57 @@ public async Task<Result> Update(PaymentIntent paymentIntent, string? message, C
             transaction.MemberId, transaction.FacilityId, transaction.Message, transaction.State, transaction.Created);
 
 
-    private static Expression<Func<Facility, FacilityResponse>> FacilityProjection()
-        => facility => new FacilityResponse(facility.Id, facility.Name, facility.Address, facility.AccountId, facility.AvatarUrl, null);
+    private static Func<IGrouping<GroupByFacility, FacilityTransaction>, FacilityTransactionResponse> FacilityTransactionResponseProjection()
```

Comment:
Looks like each of these projections use only in a single place, so they could be inlined